dek-registry-client/src/main/java/io/confluent/dekregistry/client/CachedDekRegistryClient.java

```diff
@@ -234,35 +291,88 @@ public void deleteKek(
       Map<String, String> requestProperties, String kekName, boolean permanentDelete)
       throws IOException, RestClientException {
     restService.deleteKek(requestProperties, kekName, permanentDelete);
-    kekCache.invalidate(new KekId(kekName, permanentDelete));
+    kekCache.invalidate(new KekId(kekName, false));
+    kekCache.invalidate(new KekId(kekName, true));
```

Comment:
is `new KekId(kekName, true)` ever PUT into the `kekCache`?

Reply:
Yes, on line 109, when looking up a soft-deleted KEK

Reply:
109 looks like its getting the Kek, not adding it? Am i missing something? https://github.com/confluentinc/schema-registry/blob/7131beb6882056093d6c1f5b1096df0fe68ac22f/dek-registry-client/src/main/java/io/confluent/dekregistry/client/CachedDekRegistryClient.java#L109

Reply:
kekCache is a LoadingCache, the lambda that is passed to get will be called to populate the cache if it does not exist